decidim-proposals/app/forms/decidim/proposals/admin/proposal_base_form.rb

```diff
@@ -29,8 +28,7 @@ class ProposalBaseForm < Decidim::Form
         validate :notify_missing_attachment_if_errored
 
         def map_model(model)
-          body = translated_attribute(model.body)
-          @suggested_hashtags = Decidim::ContentRenderers::HashtagRenderer.new(body).extra_hashtags.map(&:name).map(&:downcase)
+          translated_attribute(model.body)
```

Comment:
You need to leave the assignment in the `map_model`, if not it will break your code